src/item_contents.cpp

```diff
@@ -357,7 +357,8 @@ struct item_contents::item_contents_helper {
         }
 
         if( failure_messages.empty() ) {
-            return ret_val<my_pocket_type *>::make_failure( null_pocket, _( "is not a container" ) );
+            return ret_val<my_pocket_type *>::make_failure( null_pocket, _( "pocket with type (%s) not found" ),
+                    io::enum_to_string<item_pocket::pocket_type>( pk_type ) );
```

Comment:
```suggestion
                    io::enum_to_string( pk_type ) );
```
This doesn't need an explicit template argument; it will be deduced.